app/cli/cmd/attestation_add.go

```diff
@@ -90,13 +100,12 @@ func newAttestationAddCmd() *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&name, "name", "", "name of the material to be recorded")
```

Comment:
maybe be more explicit here in the description of the flag.

`name of the material as shown in the contract`